.github/workflows/quarkus-snapshot.yaml

```diff
@@ -0,0 +1,51 @@
+name: "Quarkus ecosystem CI"
```

Comment:
Why do you want to add this file to our repo? Quarkus project will not fetch our entire repository. They only want to take docs and quickstart. 
Besides that, we need to integrate with `quarkus-platform` (https://datastax-oss.atlassian.net/browse/JAVA-2708)
by creating a PR with a couple of our IT tests to their repo.

I think that in this ticket, we should take a look at the quarkus CI process and adapt our Jenkinsfile accordingly. wdyt?

Reply:
This file is referenced as item 4 in their README for adding a new extension to their CI process... see the fourth item at https://github.com/quarkusio/quarkus-ecosystem-ci.  It appears that this doc is necessary to define the build process that should be used to build and test the extension against whatever is providing Quarkus master.

I explicitly avoided doing anything in this PR related to Quarkus Platform since I assumed that would be handled in JAVA-2708.  If we keep distinct tickets for those goals (which seems to make sense) I'd argue to avoid doing anything related to Quarkus Platform here.